contrib/syslog-ng-debun

```diff
@@ -870,6 +870,16 @@ fhs_set_unix () {
 	:
 }
 
+save_config_file() {
+
+	if ${syslogngctlbin} config; then	
+		${syslogngctlbin} config -p > ${tmpdir}/syslog-ng.conf 2>&1
```

Comment:
1. I would add some different name, so people could identify the way it was created. (for example `syslog-ng.ctl.running.conf`)
2. There is a `remove_passwords_from_file` function that would be advisable to use on any config

Reply:
ok. thanks for a good suggestion.